model/endpoint_instance.go

```diff
@@ -17,6 +18,7 @@ type EndpointInstance struct {
 func (e *EndpointInstance) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	err := e.execute(r.Context())
 	if err != nil {
+		trace.SpanFromContext(r.Context()).SetAttributes(attribute.String("Error Message", err.Error()))
```

Comment:
> "Error Message"

I think there is a semantic convention for the attribute name that contains error message